soapui-testserver-api/src/test/resources/test-recipes/request-with-base64-attachment.json

```diff
@@ -0,0 +1,17 @@
+{
+  "testSteps": [
+    {
+      "type": "REST Request",
+      "method": "GET",
+      "URI": "http://google.com/",
+      "attachments" : [
+        {
+          "contentType" : "text/plain",
+          "name" : "A Name",
+          "contentId" : "An id",
+          "content": "Q29udGVudA=="
```

Comment:
Yes, most of it is from TestServer. I have just ported it to SoapUI as well. Can you please create a JIRA ticket for the missing piece?